src/CabanaPD_Solver.hpp

```diff
@@ -423,6 +437,8 @@ class SolverElastic
     std::shared_ptr<integrator_type> integrator;
     std::shared_ptr<force_type> force;
     std::shared_ptr<neighbor_type> neighbors;
+    // Optional.
+    std::shared_ptr<heat_transfer_type> heat_transfer;
```

Comment:
What is this line for?

Reply:
It stores the heat transfer object, same as all the lines above

Reply:
Same as above for adding comment for "non-optional" ones.